scripts/build/termux_step_setup_variables.sh

```diff
@@ -126,6 +126,14 @@ termux_step_setup_variables() {
 	TERMUX_PKG_NO_SHEBANG_FIX=false # if true, skip fixing shebang accordingly to TERMUX_PREFIX
 	TERMUX_PKG_IS_HASKELL_LIB=true # by default assume haskell package is lib package as most of them will always be libs.
 	TERMUX_PKG_ON_DEVICE_BUILD_NOT_SUPPORTED=false # if the package does not support compilation on a device, then this package should not be compiled on devices
+	TERMUX_PKG_SETUP_PYTHON=false # setting python to compile a package
+	TERMUX_PYTHON_VERSION=$(. $TERMUX_SCRIPTDIR/packages/python/build.sh; echo $_MAJOR_VERSION) # get the latest version of python
+	TERMUX_PYTHON_TARGET_DEPS="" # python modules to be installed via pip3
+	TERMUX_PYTHON_BUILD_DEPS="" # python modules to be installed via build-pip
+	TERMUX_PYTHON_COMMOM_DEPS="" # python modules to be installed via pip3 or build-pip
+	TERMUX_PYTHON_CROSSENV_PREFIX=$TERMUX_TOPDIR/python-crossenv-prefix # python module dependency location (only used in non-devices)
+	TERMUX_PYTHON_LDFLAGS="" # LDFLAGS which will be configured before installing python modules dependency
+	TERMUX_PYTHON_PIP_VARIAB="" # variable to be set before pip in termux_step_make_install
```

Comment:
I think all these variables except TERMUX_PYTHON_VERSION and TERMUX_PYTHON_CROSSENV_PREFIX should have a TERMUX_PKG prefix. As I see it all variables that can be set in a build recipe should start with TERMUX_PKG, while "internal variables" that are calculated somehow (like TERMUX_PYTHON_VERSION), use just TERMUX